community/cypher/cypher/src/main/java/org/neo4j/cypher/export/SubGraphExporter.java

```diff
@@ -90,12 +90,13 @@ private void output( PrintWriter out, String ... commands )
                 {
                     throw new IllegalStateException( "Indexes should have at least one property key" );
                 }
-                String key = quote( propertyKeys.next() );
+                String id = propertyKeys.next();
                 if ( propertyKeys.hasNext() )
                 {
                     throw new RuntimeException( "Exporting compound indexes is not implemented yet" );
                 }
 
+                String key = quote( id );
```

Comment:
I don't understand what this improves. Sure, it doesn't hurt, but I'm curious to learn what it is supposed to improve.

Reply:
The reason for that was "Move declaration closer to actual usage". Its usefulness is debatable, you are right. Say the word and I kill it